.envrc

```diff
@@ -0,0 +1 @@
+source_up
```

Comment:
I don't think that we need this?

Reply:
Oh yeah you're right. Why is this file even checked in? I'll get rid of it